pandas/conftest.py

```diff
@@ -953,3 +954,69 @@ def __len__(self):
             return self._data.__len__()
 
     return TestNonDictMapping
+
+
```

Comment:
can you do 'sections' here e.g. some text markers so reading the file is easy (@jbrockmendel has added these in several files); this can be a followup PR.